### PR TITLE
Setup Gemini and write test for all pages

### PR DESCRIPTION
**Pull request by NazarKY**
Review tools some of them in http://habrahabr.ru/company/yandex/blog/238323/
http://tldr.huddle.com/blog/css-testing/
setup gemini or similar tool and base screenshots
Base browser display sizes are: 1440x900, 1920x1080, 1366x768

### CODE REVIEW

File: tests/gemini_tests/all_pages.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var gemini = require('gemini'),
+  glob = require('glob');
+
+gemini.suite('pages', function(suite) {
+  pages_arr = glob.sync('./_site/**/*.html')
```
Comment (pftg) [inline tests/gemini_tests/all_pages.js]: please remove `_arr` suffix

**Comment by pftg** [comment]
do not see saved screens

File: .gemini.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+rootUrl: http://localhost:4000/
+screenshotsDir: gemini/screens
```
Comment (pftg) [inline .gemini.yml]: use `reports/screenshots` as folder

**Comment by pftg** [comment]
- [x] upload latest master base screenshots
- [x] show failed reports
- [x] show success reports

**Comment by NazarKY** [comment]
- Have upload latest images. Can get by this [link](https://drive.google.com/folderview?id=0B9jNBsx5RWYGb0Q2SmRLVDJYeUE&usp=sharing). And added page on [wiki](https://github.com/jetthoughts/jetthoughts.github.io/wiki/ScreenShots-of-original-pages)

**Comment by NazarKY** [comment]
- Have uploaded failed reports for all blogs (not for all site) - can get by [link](https://drive.google.com/folderview?id=0B9jNBsx5RWYGYXYxN2M5MmVJOFE&usp=sharing)

**Comment by NazarKY** [comment]
- With success reports is another issue: gemini haven't give any success report - it always failed on fonts in footer.
  Need extra investigation with it. But you can still watch the report by the [link](https://drive.google.com/folderview?id=0B9jNBsx5RWYGOW5xZlc3VlNrM2s&usp=sharing)

**Comment by pftg** [comment]
@NazarKY please paste here link to gist with error message and screenshot

File: tests/gemini_tests/all_pages.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+var TAIL = 4
```
Comment (pftg) [inline tests/gemini_tests/all_pages.js]: not descriptive variable name: `HTML_EXTENSION_SIZE`

**Comment by NazarKY** [comment]
@pftg: pls review

File: tests/gemini_tests/all_pages.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+var screens = {
+    big: {width: 1920, height: 1080}, meddium: {width: 1440, height: 900},
+    small: {width: 1366, height: 768}, mobile: {width: 400, height: 768}
+  };
+
+for(var i = 0; i < pages_arr.length; i++) {
+  page_path = pages_arr[i].replace('./_site', '');
+  page_name = page_path.replace(/[^\w\s]/gi, '');
+  gemini.suite('1366x768_' + page_name.substring(0, page_name.length - TAIL), function(suite) {
```
Comment (pftg) [inline tests/gemini_tests/all_pages.js]: extract `page_name.substring(0, page_name.length - TAIL)` in variable

**Comment by pftg** [comment]
@NazarKY do not forget squash commits

**Comment by NazarKY** [comment]
@pftg: pls review

**Comment by pftg** [comment]
- [x] update README how to use gemini in the project: how to gather base version and how to test on localhost and how to test on another url

**Comment by NazarKY** [comment]
@pftg: pls review

**Comment by pftg** [comment]
@tehhi42 please check english in readme, and comment all errors

**Comment by tehhi42** [comment]
@pftg @NazarKY done.

**Pull request merged**